app/services/new_schedule_builder.rb

```diff
@@ -15,4 +17,4 @@ def self.create_empty_schedule
       end
     end
   end
-end
\ No newline at end of file
+end
```

Comment:
Yeah, this was interesting to me. I guess most of the files we work with for instructor apps don't have the extra line at the end.